community/gardening/developing-plugins.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: single
+title:  "Plugin Development at Spinnaker Gardening Days"
+sidebar:
+  nav: community
+---
+The Spinnaker plugin framework simplifies loading and managing third-party extensions. This makes it easier to extend and integrate with Spinnaker, and to share and evolve those extensions throughout their lifecycles. At this stage, the new plugin framework has a growing list of well-defined extension points. It's time for the community to begin leveraging those extension points to add functionality to Spinnaker, and to explore development of new extension points.
+
+## Intro to plugins assignment at Spinnaker Gardening Days
+
+To jump into Spinnaker plugins, start by creating a custom separate stage plugin. This will help you become familiar with using the framework. For your first plugin project, we recommend that you leverage the new SimpleStage extension point to add your custom pipeline stage to Spinnaker. Read more in the [Plugin Creators Guide](https://www.spinnaker.io/guides/developer/plugin-creators/).
```

Comment:
Is "separate" meaningful/necessary here? Could we just say "Create a custom-stage plugin and..."
And there are several other occurrences in this PR.

Reply:
No I think you're right, it's confusing and not necessary. Thx

Reply:
I only found 1 occurrence of the word "separate." (removed). Did you find other wordiness issues @dorbin ?

Reply:
I just marked the other one.

Reply:
Sorry sorry, duh. Multitasking bad

Reply:
thank you @dorbin